src/Browscap/Generator/DiffGenerator.php

```diff
@@ -108,7 +108,7 @@ public function run($leftFilename, $rightFilename)
             $msg = sprintf(
                 '%sThere %s %d difference%s found in the comparison.',
                 "\n",
-                ($this->diffsFound === 1 ? 'was'  : 'were'),
+                ($this->diffsFound === 1 ? 'was' : 'were'),
```

Comment:
I ran `./vendor/bin/php-cs-fixer fix` after I made my changes, and this file was modified.  Probably missed in earlier php-cs-fixer runs.